cmd/package_test.go

```diff
@@ -64,8 +64,6 @@ var allowedCalls = map[string]set.Strings{
 	"juju/commands/upgrademodel.go": set.NewStrings("os.Open", "os.RemoveAll"),
 	// ssh is not a whitelisted embedded CLI command.
 	"juju/ssh/ssh_machine.go": set.NewStrings("os.Remove"),
-	// agree is not a whitelisted embedded CLI command.
-	"juju/romulus/agree/agree.go": set.NewStrings("exec.Command", "exec.LookPath"),
```

Comment:
rather than just removing this, should it be moved to "juju/agree/agree.go"?

It might be nice if we didn't have to explicitly except agree, but it seems that Agree wants to use $PAGER to provide the terms with a scrollable dialog, and so probably does still need to be included here.

Please fix up the comment, as it is clearly just copy/pasted from ssh. Instead it should say "agree is not exposed to shell scripts because it uses PAGER to present terms", or something to that effect.
We should also make sure that `juju shell` doesn't let you use agree from the web browser.

Reply:
Will do